src/components/TopBar/TopBar.vue

```diff
@@ -103,13 +103,19 @@
 				:icon="iconMenuPeople"
 				@click="openSidebar" />
 		</Actions>
+		<AppNavigationCounter
```

Comment:
this is abuse and might break without previous note as AppNavigation* should only be used in AppNavigation

Reply:
yeah, was afraid of that... so maybe we first adjust the nextcloud-vue component to be more generic.
might be better than just copy-pasting all the styles over here

Reply:
now thinking, a generic CounterBubble might not be good either because it would always need styling hacks to position it.

@ma12-co would it make sense to introduce a ActionButtonCounter, or a property for action buttons to display a bubble nearby ?

Reply:
Sorry to jump in here but this would maybe also be a nice feature for other apps (see e.g. https://github.com/nextcloud/spreed/issues/5525 )

Reply:
yeah, so generic component it is: https://github.com/nextcloud/nextcloud-vue/pull/1895

Reply:
adjusted to use CounterBubble now